infrastructure/keepalived_reload.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "#### new keepalived config ####"
+cat /etc/keepalived/keepalived.conf
+service keepalived restart || true
```

Comment:
no, this script is trigger by `consul-template` similar to `haproxy_reload.sh` ...